### ISSUE TITLE
cannot stopPropagation.

### ISSUE THREAD

**Issue by sparrow1116** [hammerjs/hammer.js#1175]
I have a case with the dom like below
<div>
          <img/>
</div>
<div>
        <img/>
<div>
I want to drag img to any position.  and I also want to drag div.
when i  panstart on img. i just want to drag the img.   and when my panstart on div.  the div will be drag.

so I add stopPropagation on img. but it cannot work.

**Comment by weituotian** [comment]
+1, i also encounter this issue, and want to know how to do it

**Comment by weituotian** [comment]
@hammerjs 
if child  stopPropagation, the parent always fire the events

**Comment by TheDutchCoder** [comment]
Yeah this has still not been fixed on mobile. It still fires off 2 events and you can't stop their propagation :(

**Comment by capodisseny** [comment]
still happening? any solution? thanks

**Comment by weituotian** [comment]
好~(￣▽￣)~*
